apps/src/templates/instructions/RubricField.jsx

```diff
@@ -2,6 +2,7 @@ import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import color from '@cdo/apps/util/color';
 import Radium from 'radium';
+import {CheckRadioButton} from '../../lib/ui/CheckedRadioButton';
```

Comment:
nit: keep the naming consistent for the component:

```suggestion
import {CheckedRadioButton} from '../../lib/ui/CheckedRadioButton';
```